react/features/base/media/actions.js

```diff
@@ -97,6 +102,7 @@ export function setVideoAvailable(available: boolean) {
  */
 export function setVideoMuted(
         muted: boolean,
+        mediaType: MEDIA_TYPE = MEDIA_TYPE.VIDEO,
```

Comment:
Why does this need to be explicit? I think this action could figure out what to do based on the current track state, can't it?

Reply:
As discussed with you, we would need this to mute either of the presenter or video tracks explicitly.